feat(elastic-metal): updated proxmox cluster private networks tutorial MTA-7321 (#6711)
scaleway/docs-content: tutorials/setup-elastic-metal-proxmox-cluster-with-private-networks/index.mdx

  Repeat these steps for each Elastic Metal Server you want to add to your cluster.
</Message>

> 1. Select your Elastic Metal server by clicking on **Elastic Metal** in the **Bare Metal** section of the side menu of the Scaleway console.

Reviewer: ```suggestion
1. In the **Bare Metal** section of the console side menu, click **Elastic Metal** to select your Elastic Metal server.
```

> 2. Choose the Proxmox-compatible server you want to install from the list and click its name to see the server dashboard.

Reviewer: ```suggestion
2. From the list, select the Proxmox-compatible server you want to install and click its name to open the server dashboard.
```

3. Select **Proxmox** from the list of available images.
    <Lightbox image={image} alt="" />

> 4. Enter your credentials. The service username is `root`. The service password must be at least 8-character long.

Reviewer 2: ```suggestion
4. Enter your credentials. The service username is `root`. The service password must be at least eight characters long.
```

Reviewer 3: "two" was changed to "2" on line 36. And now "8" was changed to "eight".

Reviewer 2: Justification for my suggestion concerning "eight" (my suggestion does not concern "two" or "2", must be coming from someone else):
- "...must be at least 8-character long" is not correct grammatically --> "an eight-character-long password" versus "a password that is eight characters long"
- https://www.scaleway.com/en/docs/guidelines/#numbers --> Write numbers one to nine in full

Author: Thank you guys, we'll talk about the number rule with the guidelines ^^

Once the server has rebooted, the Proxmox web interface is available.

1. Open a web browser and go to `https://your_elastic_metal_server_ip:8006/`. The Proxmox login screen opens.
2. Enter your credentials and click **Login**.

    Enter the credentials to log in:

      - **User name**: `root`
      - **Password**: The password set during the installation of Proxmox
      - **Realm**: `Linux PAM standard authentication`
      - **Language**: The display language of the Proxmox interface (i.e., `English`)

    The Proxmox dashboard displays.

## Configuring the Private Network

> You must configure the Private Network interface using the Proxmox interface (rather than [from your server's command line](/elastic-metal/how-to/use-private-networks/#how-to-configure-the-network-interface-on-your-elastic-metal-server-for-private-networks)).

Reviewer: ```suggestion
You must configure the Private Network interface from the Proxmox interface, not [from your server's command line](/elastic-metal/how-to/use-private-networks/#how-to-configure-the-network-interface-on-your-elastic-metal-server-for-private-networks).
```

> 1. Click the Proxmox host in the data center menu on the left. The server's dashboard appears.

Reviewer: ```suggestion
1. In the **Data center** menu on the left, click the Proxmox host. The server's dashboard appears.
```

> 2. Click **Network** in the **System** section of the menu. The network dashboard appears.

Reviewer: ```suggestion
2. In the **System** section of the menu, click **Network**. The network dashboard appears.
```

## Creating a cluster

> 1. Click **Cluster** in the data center view of the Proxmox interface of your first node. The cluster dashboard displays.

Reviewer: ```suggestion
1. In the data center view of the Proxmox interface of your first node, click **Cluster**. The cluster dashboard displays.
```

8. Enter your first node's `root` password and click **Join** to link the machine to the cluster.
    <Lightbox image={image7} alt="" />

> The second node now appears in the cluster in the Data center view of Proxmox. You can manage both machines using a single interface. The communication between members of the cluster is entirely in your Private Network, ensuring the highest level of security and the lowest latency.

Reviewer: data center view appears several times, sometimes capitalized and sometimes not.

Author: Yes, I've noticed, but since I don't know what the Proxmox interface looks like (yet), I've decided to leave this point aside for now. I'll ask Alexia!
